integration_test.go

```diff
@@ -589,3 +626,92 @@ func generateData(numRows int, sleepTime time.Duration, sendBadRow bool) error {
 
 	return nil
 }
+
+// Log entry template with all bigquery fields
+type log_entry_alltypes struct {
+	StringField     string  `json:"StringField"`
+	BytesField      []byte  `json:"BytesField"`
+	IntegerField    int64   `json:"IntegerField"`
+	FloatField      float64 `json:"FloatField"`
+	NumericField    string  `json:"NumericField"`
+	BigNumericField string  `json:"BigNumericField"`
+	BooleanField    bool    `json:"BooleanField"`
+	TimestampField  int64   `json:"TimestampField"`
+	DateField       int32   `json:"DateField"`
+	TimeField       string  `json:"TimeField"`
+	DateTimeField   string  `json:"DateTimeField"`
+	GeographyField  string  `json:"GeographyField"`
+	RecordField     struct {
```

Comment:
Can you add another sub field to the record with a type that will verify the recursive call to `mangleInputSchema` with a changed type, such as `DateTime` or `Numeric`?

Reply:
Yes this is already done: https://github.com/raahi-chada2004/out_writeapi/blob/eb3ff4a8d2068a6329d0fff1499d8052c903d3b9/integration_test.go#L83 - the second subfield is of type numeric (and the data input is string for numeric type).

Reply:
Oh, I missed it. Thanks for clarifying!